includes/Admin.php

```
@@ -96,7 +96,7 @@ public function enqueue_admin_scripts( $hook ): void {
 		wp_register_script(
 			'salt-shaker-admin',
 			SALT_SHAKER_PLUGIN_URL . 'assets/build/admin.js',
```

PHPStan (2.2.5)

[error] Constant SALT_SHAKER_PLUGIN_URL not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(constant.notFound)